kotskinds/apis/kots/v1beta1/helmchart_types.go

```diff
@@ -40,6 +40,15 @@ type MappedChartValue struct {
 	array    []*MappedChartValue          `json:"-"`
 }
 
+func (m MappedChartValue) MarshalJSON() ([]byte, error) {
+	val, err := m.getBuiltValue()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get built value")
+	}
+
+	return json.Marshal(val)
+}
+
```

Comment:
this is the only change to the apis, so I'm not sure why the `additionalProperties` on the helmchart schema are being generated.